en/messages.json

```diff
@@ -245,6 +245,32 @@
         "message": "Firefox Relay can now forward emails up to 25MB, including attachments.",
         "description": "DO NOT TRANSLATE \"Firefox Relay\"."
     },
+    "popupEndOfIntroPricingHeadline": {
+        "message": "Our intro pricing offer ends soon!"
+    },
+    "popupEndOfIntroPricingBody": {
+        "message": "Get Relay Premium before September 27th and enjoy unlimited masking at our introductory month-to-month price.",
+        "description": "DO NOT TRANSLATE \"Relay Premium\"."
+    },
+    "popupEndOfIntroPricingCTA": {
+        "message": "Upgrade now"
+    },
+    "popupEndOfIntroPricingCountdownDays": {
+        "message": "Days",
+        "description": "This label displayed on top of a large number representing the number of DAYS of the remaining time the introductory pricing offer is still valid. Please abbreviate if the word exceeds 5 characters."
```

Comment:
There is a grammar error: need to add `that` in front of `the introductory pricing`. Same issue with the following descriptions.

Reply:
Resolved. Let me know if this is good to merge!